node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/component/ConfigServerInfo.java

```diff
@@ -86,6 +93,18 @@ public URI getConfigServerUri(String hostname) {
                 hostname -> URI.create(scheme + "://" + hostname + ":" + port)));
     }
 
+    private static ConfigServerConfig.Sia verifySiaConfig(ConfigServerConfig.Sia sia) {
+        List<String> configParams = Arrays.asList(
+                sia.credentialsPath(), sia.configserverIdentityName(), sia.hostIdentityName(), sia.trustStoreFile());
```

Comment:
the configserverIdentityName is actually not related SIA? I thought one of the conclusions from yesterday's discussion was that SIA provides exactly one identity (=hostIdentityName).

Reply:
It's not related to SIA per se, but it is required for constructing a `HostnameVerifier` that accepts the certificate from configserver. I agree that it's somewhat misleading to have the parameter under `Sia`.